Runtime/Scripts/InstanceRenderingInfo.cs

```diff
@@ -0,0 +1,238 @@
+﻿using System;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Rendering;
+
+namespace Unity.MeshSync
+{
+    public class InstanceRenderingInfo
+    {
+        public Mesh Mesh;
+        
+        public List<Matrix4x4[]> DividedInstances { get; private set; } = new List<Matrix4x4[]>();
+
+        private Material[] m_materials;
```

Comment:
Shouldn't `m_materials` be serialized ?

Reply:
There was an iteration where InstanceRenderingInfo was serializable but we have moved away from that